tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,4 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == False
```

Comment:
Great start! It would also be good to check that the length of the inventory has not changed and that nothing was accidentally removed.